pages/platform/private-registry/configure-oidc-provider-authentication/guide.en-au.md

```diff
@@ -0,0 +1,288 @@
+updated: 2023-08-25
+---
+
+<style>
+ pre {
+     font-size: 14px;
+ }
+ pre.console {
+   background-color: #300A24; 
+   color: #ccc;
+   font-family: monospace;
+   padding: 5px;
+   margin-bottom: 5px;
+ }
+ pre.console code {
+   border: solid 0px transparent;
+   color: #ccc;
+   font-family: monospace !important;
+   font-size: 0.75em;
+ }
+ .small {
+     font-size: 0.75em;
+ }
+</style>
+
+## Objective
+
+The OVHcloud Managed Private Registry service, a cloud-native registry built on Harbor, allows you to store, manage and access your container images (OCI artifacts) and Helm charts.
+
+By default, to log in to your OVHloud Managed private registries you have to generate credentials (from the OVHcloud Control Panel, the APIv6 or Terraform), then log in with this user and administrate other users in the Harbor HMI.
+
+However, you can also configure an Open ID Connect (OIDC) provider, for the OVHcloud Managed Private registry authentication, such as Keycloack.
+
+- First of all, what is OIDC?
+
+![OIDC](images/oidc.png)
+
+OIDC stands for [OpenID Connect](https://en.wikipedia.org/wiki/OpenID). It is an open standard and decentralized authentication protocol.
+This protocol allows verifying the user identity when a user is trying to access a protected HTTPs endpoint.
+
+Several OpenID Connect providers exists like Dex, Keycloak, Okta or a SaaS provider. Harbor supports all of them. In this tutorial we will use Keycloack.
+
+- What is Keycloak?
+
+![Keycloak](images/keycloack.png)
+
+[Keycloak](https://www.keycloak.org/) is an open source Identity and Access Management solution aimed at modern applications and services.
+It makes it easy to secure applications and services with little to no code.
+More information can be found here: [Official Keycloak documentation](https://www.keycloak.org/documentation.html).
+
+**This guide will explain how to configure the authentication of your OVHcloud Managed Private Registry service to use an OIDC provider.**
+
+## Requirements
+
+- An OVHcloud [Managed Private Registry](/pages/platform/private-registry/creating-a-private-registry).
+- The URL and login/password of your private registry.
+- an OIDC provider, like a KeyCloack instance and credentials to access it.
+
+Follow the official guide if you want to [install Keycloack on an OVHcloud Managed Kubernetes cluster](/pages/platform/kubernetes-k8s/installing-keycloak).
+
+## Instructions
+
+### Configure ythe OIDC provider through the API
+
+#### The API Explorer
+
+To simplify things, we are using the [API Explorer](https://api.ovh.com/) which allows to explore, learn and interact with the API in an interactive way.
+
+Log in to the API Explorer using your OVHcloud account credentials.
+
+![Log in to the API Explorer](images/kubernetes-quickstart-api-ovh-com-001.png){.thumbnail}
+
+If you go to the [Kubernetes section](https://api.ovh.com/console/#/cloud/project/%7bservicename%7d/kube~get) of the API Explorer, you will see the available endpoints:
```

Comment:
In the section we are talking about the Kubernetes section, shouldn't we talk about Registry section  ?